deeprankcore/trainer.py

```diff
@@ -42,14 +42,14 @@ def __init__( # pylint: disable=too-many-arguments, too-many-branches
                 in terms of output shape (Trainer class takes care of formatting the output shape according to the task).
                 More specifically, in classification task cases, softmax shouldn't be used as the last activation function.
 
-            dataset_train (GraphDataset object, optional): training set used during training.
-                Can't be None if pretrained_model is also None. Defaults to None.
+            dataset_train (deeprank dataset object, optional): training set used during training.
+                Can't be None if pretrained_model_path is also None. Defaults to None.
 
-            dataset_val (GraphDataset object, optional): evaluation set used during training.
+            dataset_val (deeprank dataset object, optional): evaluation set used during training.
```

Comment:
```suggestion
            dataset_val (deeprank-core dataset object, optional): evaluation set used during training.
```

Reply:
Changed this in commit 9ad784c964ef77876f71e39599c150dcace6bb22.